conftest.py

```diff
@@ -677,17 +679,22 @@ def setup_test_colo(
         exp: Experiment,
         db_args: t.Dict[str, t.Any],
         colo_settings: t.Optional[t.Dict[str, t.Any]] = None,
+        on_wlm: t.Optional[bool] = False
     ) -> Model:
         """Setup things needed for setting up the colo pinning tests"""
         # get test setup
-        test_dir = fileutils.make_test_dir(level=2)
+        level = 3 if on_wlm else 2
+        test_dir = fileutils.make_test_dir(level=level)
```

Comment:
Could you recheck this stack level? When I was running stubs of the `on_wlm/test_colocated_model` tests on horizon and I had the following path returned for the `test_dir`
```
test_dir='/path/to/my/clone/of/smartsim/tests/test_output/../../../../../../path/to/my/env/ss/lib/python3.9/site-packages/_pytest/python/pytest_pyfunc_call'
```

Reply:
Eyeballing the current implementation of the tests, I cannot immediately see where the stack level would need be different between the `test_colo_model_local.py` tests and the `on_wlm/test_colocated_model.py` tests.

Was this change necessitated by change in machine/virtualenv provider?